css/css-writing-modes/forms/textarea-appearance-native-horizontal.optional.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<link rel="author" title="Tim Nguyen" href="https://github.com/nt1m">
+<link rel="help" href="https://html.spec.whatwg.org/#the-textarea-element">
+<link rel="help" href="https://drafts.csswg.org/css-writing-modes-4/#block-flow">
+<title>native textarea in horizontal writing mode</title>
+<meta charset="utf-8">
+<link rel="mismatch" href="textarea-appearance-native-vlr.optional.html">
+<link rel="mismatch" href="textarea-appearance-none-vlr.optional.html">
+<link rel="mismatch" href="textarea-appearance-native-vrl.optional.html">
+<link rel="mismatch" href="textarea-appearance-none-vrl.optional.html">
```

Comment:
Why not also mismatch with appearance-none horizontal? (And why are the sideways values not tested?)

Reply:
> Why not also mismatch with appearance-none horizontal?

I think this is out of scope of css-writing-modes, probably would made more sense if it was for css-ui.

> (And why are the sideways values not tested?)

Mostly to keep these reftests self contained for Interop 2023. I could create a separate PR for those later on (that covers the other form controls already in the directory too).